app/components/floating_text_input_label/index.tsx

```diff
@@ -19,11 +19,14 @@ const BORDER_FOCUSED_WIDTH = 2;
 
 const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     container: {
-        height: DEFAULT_INPUT_HEIGHT + (2 * BORDER_DEFAULT_WIDTH),
         width: '100%',
     },
     errorContainer: {
         flexDirection: 'row',
+
+        // Hack to properly place text in flexbox
+        borderColor: 'transparent',
+        borderWidth: 1,
```

Comment:
The problem is that, the way flexboxes and textboxes work, the error message was overflowing outside of the error container. This was probably not noticeable in most error boxes since the error rarely had more than one line.

The border is (not sure why) forcing the text to be contained within the error container.